layers/+emacs/org/README.org

```diff
@@ -291,6 +293,23 @@ To permanently enable mode line display of org clock, add this snippet to your
   (setq spaceline-org-clock-p t)
 #+END_SRC
 
+** Jira support
+To bring Jira and OrgMode together over [[https://github.com/ahungry/org-jira][org-jira]] set the variable =org-enable-jira-support= to =t=.
+
+#+BEGIN_SRC emacs-lisp
+  (setq-default dotspacemacs-configuration-layers '(
+    (org :variables
+         org-enable-jira-support t
+         jiralib-url "https://yourcompany.atlassian.net:443")))
+#+END_SRC
+
+If you would like to avoid being prompted for your login and password each time you connect,
+  add your authentication credentials to =~/.authinfo.gpg= or =~/.authinfo=:
+
+#+BEGIN_SRC
```

Comment:
The build is failing because you don't have a language specified in this `BEGIN_SRC` block.

Reply:
@sdwolfz can you suggest a language for `authinfo`? I've checked https://orgmode.org/manual/Languages.html and haven't got any idea.

Reply:
I think you can safely put in `authinfo`. It does not necessarily need to be a language for which emacs has a mode available.

Reply:
@sdwolfz I see. Pushed.